go.mod

```diff
@@ -31,9 +31,11 @@ require (
 )
 
 // Workaround for https://github.com/golang/go/issues/30831 and fallout.
-replace github.com/golang/lint => github.com/golang/lint v0.0.0-20190227174305-8f45f776aaf1
+replace github.com/golang/lint => github.com/golang/lint v0.0.0-20181217174547-8f45f776aaf1
```

Comment:
I think some these might be able to be removed. Have you tried that?

Reply:
Good idea. Tried it, here's what's required now.